### DISCUSSION TITLE
a brief history of the galaxy; Michael Zahniser's speculative lore document

### DISCUSSION THREAD

**Discussion by bene-dictator** [endless-sky/endless-sky#9960] [Lore]
**A link to the Google Document where the text below originates:**
https://docs.google.com/document/d/1CpK_ol0E7a86LmEneFIJm1B9U-t357YQOZR-1OvYgcc/edit

This document was published on 25 Dec 2020 by the creator of Endless Sky, Michael Zahniser. I am posting it here on the repository's discussions page as it is more visible here than at the bottom of the pins on ESC/content-creating. Furthermore, it provides a lasting place to talk about and question the document.

As its introductory note states, this lore document is _not the decided and definite_ canon of the game, and furthermore parts of it have been refuted or replaced ingame as the community and development team create the stories of Endless Sky (notably, the secret Sheragi colony, and much of the things to do with the Coalition and Heliarchy). Nonetheless, the document is still valuable and provides insights into how MZ developed the species and stories of the game we know (and perhaps love).

Below lies the full, unaltered text.

<hr>

## Introductory note
It’s worth keeping in mind that pretty much every species or faction in Endless Sky lies to you, either because they have their own agenda or because they don’t know what the heck they’re talking about (or both). Sifting out what’s actually true is part of the fun, and my intention was to have different people tell you obviously contradictory stories about the same things. (This also creates the degree of moral ambiguity necessary for the player to take different sides in various conflicts without feeling like there’s a “good side” and a “bad side.”) But this is more or less my outline of what the “real” true history is.

Also note that some of this lore is more fully-formed than others. For example, most of the Coalition lore isn’t yet reflected in-game, and is more of just a sketch of where I thought that story line might go. If any of this contradicts your head-canon and your head-canon is a cooler story than what I’ve come up with, don’t feel like you need to throw out your ideas in favor of mine. :)

## Cosmology
Spacetime is composed of two layers which the Quarg refer to as the “warp” and the “weft.” (This is a textile analogy.)

The warp is the ordinary, linear spacetime that human beings, etc. inhabit. Travel within the warp is limited to the speed of light, but technology can be used to build paths between any two sufficiently massive objects (e.g. stars) where the distance is artificially condensed. These paths are what humans refer to as “hyperspace lanes,” although most humans believe them to be natural phenomena. Building these links requires matched quantum-linked transmitters, so you can’t build a link to a new star system without first sending a transmitter there by other means (either slower-than-light travel or jump drive).

The weft, on the other hand, is a dimension where space and time are nonlinear. By briefly passing through weft-space, a ship can instantaneously travel a significant distance in linear space. The jump drive is the technology that allows this sort of travel. It relies on exotic technology that few species can replicate and fewer still understand. For example, the Korath exiles (tier 2.5) are able to build jump drives in limited numbers after reverse-engineering Quarg technology, but they suffer frequent destructive manufacturing mishaps.

Hyperspace (warp) travel, on the other hand, is accessible to a tier 1 civilization, but the technology to create hyperspace lanes is much more advanced, roughly tier 3; it’s more or less a matter of luck whether a species discovers it before discovering the jump drive. The Quarg, for example, never discovered how to build hyperspace lanes. (But, the Archons are capable of constructing them.)

For a very long time, advanced life did not evolve in most star systems because the early planetary systems did not contain enough of the heavier elements. It was only after several generations of stars began to go nova, seeding the galaxy with the elements produced in their cores through nuclear fusion, that life even became possible...

## The Drak
The Drak were the first advanced civilization to arise in our galaxy, about 135 million years ago. Because no other civilizations had come before them, there were not yet any hyperspace lanes; this was also long before the Pug began to be involved here. As a result, their civilization grew very slowly through slower-than-light travel, and lasted for hundreds of thousands of years, but in many divergent pockets rather than one unified government. 

Eventually they discovered the means to build hyperspace links between their star systems, but rather than ushering in a new revolution of technology and civilization it led to thousands of years of war as the extremely different cultures that had splintered off from the Drak homeworld came into contact with each other. Eventually by mutual decision they severed the hyperlinks between the factions and settled into an uneasy peace...

...at least until they discovered the jump drive, and there was no longer any way to set up artificial barriers between themselves. This led to another round of civil wars that were far more devastating, with weapons that turned entire sections of the galaxy permanently fallow (e.g. the Ember Waste). It seemed likely that the Drak would drive themselves extinct. At this point, several different leaders among them proposed different ways to end the wars.

One group was fascinated by the discovery of the weft-space, believing it to be a higher plane of reality. They wanted to build the technology to explore weft-space further, hoping that in its complex and non-linear reality a species could transcend the limitations of our reality and develop untold power. They had already seen promising results through the construction of technology, such as the jump drive, that exists partly in warp-space and partly in weft-space. But what more would they be able to accomplish if they could inhabit the weft instead of just passing through it? Surely, a grand project like that could draw all the warring factions together.

A second group was more concerned with simply ending the civil wars. They became convinced that the only solution was to build something akin to artificial gods, who would be tasked with mediating peace and preventing the Drak’s extinction. They began experimenting with genetic modifications of a space-dwelling life form (the void sprites), melding biology and technology together with weft-space exotic matter. This project was done in secret, because they feared (rightly) that the rest of Drak civilization would believe they were building some sort of super-weapon. (The original Archon design document states, “Archons are made partly of biological components, partly of technological ones, and partly of components for which no descriptive category exists in human language. When upset, they vomit antimatter.”)

## The great disappointment
The weft-space mania swept through Drak civilization. They believed that there was a miraculous reality available to them, and that the destiny of their entire species was to transcend this mundane reality and explore, and eventually conquer, the weft. Their whole species would need to enter the weft together, because the journey would change them, turning them into something beyond anything they could presently imagine; any Drak who stayed behind would be nothing but a lesser species when the evolved Drak returned.

It became a sort of religion among them, and did indeed quiet the wars for some time… until their explorations began to produce results. The lead scientists soon came to realize that the weft, despite its exotic complexity, was completely barren, a place where life could not develop or thrive. They had hoped for a paradise, a higher level of existence, and found instead a chaotic void. Many of the Drak, however, chose not to believe this.

Meanwhile, the Archon-builders continued their work in secret.

The wars returned, more devastating than before, and hopelessness swept their civilization. Eventually all hope seemed lost. The Archon-builders, who up until that point had been wary of the risks involved in creating new life forms more powerful than themselves, decided it was time. Although the Archons are living creatures and not ships, there is space within each one to hold a couple hundred Drak in stasis pods, with their memories and consciousness guiding that particular Archon. Each Archon was filled with a collection of Drak from each of their factions so that they would not prefer one over the other.

But that project, as well, was a failure. The rest of the Drak saw the Archons as enemies, and the Archons had to retreat rather than fight back and add to the bloodshed. They were able to do nothing as they watched their species drive itself extinct.
For sixty million years, the Archons were alone in the galaxy.

## The Quarg
The Quarg were the next species in the galaxy to develop to the point of discovering space travel, and the Archons made contact with them almost immediately. The Archons did not particularly wish to share with the Quarg the story of their failure to prevent the extinction of their own creators. Instead, they told the Quarg a story that went something like this:

The Drak are driven by a need to explore and to discover new things. Their civilization grew to fill the whole galaxy, and they came to understand all the laws that govern this reality. When there was nothing left to learn or discover, their civilization chose to travel through hidden paths into a deeper reality that lies beyond this one. They chose to all travel together because the journey would change them, and if any remained behind they would be nothing but a lesser species when the evolved Drak returned. Before they departed, they created the Archons as their representatives, to tend the galaxy and promote peace within it until the Drak finish their long journey and come back. The Drak now speak only through the Archons.

Perhaps because, from their earliest days, the Quarg were overshadowed by these vastly more advanced beings, their civilization developed very differently. The Archons would not allow any serious conflict to arise between them, and the Quarg came to see themselves as partners in the noble mission of the Archons to prevent future civilizations from driving themselves extinct.
The process of stellar evolution continued, resulting in more and more solar systems and planets that contained the elements necessary for life. The Archons had become convinced that the sheer size of the Drak civilization was what had caused it to splinter and collapse. At the same time, a certain degree of expansion was necessary to fuel technological advances. As the first few post-Quarg species developed interstellar civilizations, the Archons and the Quarg together tried to intervene and find the constraints - the proper number of star systems, the right interaction or lack thereof with the Quarg or with the Archons themselves - that would guarantee the survival of a species. They did not always succeed at this.

## The Pug
During this period of time, the first Pug factions came to our galaxy. Each “hive” of Pug (anywhere from 1-10 billion individuals, typically) operates independently from the others, pursuing its own goals and avoiding stepping on the toes of other Pug working on other “projects.” The Pug, like the Archons, believe that they are working to preserve other species from extinction, but they use very different methods, and from the beginning the Pug and the Quarg were at war with each other.

It is, of course, an unequal war, because the Pug are far more advanced than the Quarg. On the other hand, the Pug refuse on principle to even risk harming the Archons. Archons are built, not born; they do not reproduce, and their builders are long gone, so an Archon that is killed can never be replaced. The Pug were not willing to do something so permanent.

The result, at present, is an uneasy truce, where the Pug and the Quarg both court species as they develop. For example, the Pug visited pre-modern Earth at various times, and are the source of many fairy-folk legends. They also like to create “backups” of pre-spaceflight species, abducting large numbers of them and bringing them to a different planet just in case the rest of that species drives itself extinct while still limited to a single planet. (The human settlements in the Deep were created by the Pug in this way, with humans abducted from prehistoric times up through the middle ages.)

This also happens to mean that the Pug, in their war against the Quarg, have one final trump card they have not yet played. Although the Quarg and the Archons believe that the Sheragi are extinct, the Pug have in fact preserved a remnant of them in another galaxy. These Sheragi, of course, are thoroughly loyal to the Pug.

The Pug have baby-blue blood, like horseshoe crabs (based on hemocyanin instead of hemoglobin).
The Pug have different ships and weapons that they use depending on whether they want to engage in “fair” combat with species at various technological tiers or simply intimidate them. The Arfectas and their weapons, for example, are a Tier 3 technology, the sort of thing they would use for fighting the Quarg. The tiers go up to at least 5, with the Archons (and the civilization that created them, before it disappeared) at tier 4. In fact, the entire tier classification system is a Pug concept. Intergalactic wormholes are a tier-5 technology.

## The civilization boom
The period of time from approximately 120,000 years ago until the present has been marked by a sudden boom of new space-faring species throughout the galaxy. The boom began with the Hai. The original territory allotted to them by the Archons included about a third of human space. Hai civilization grew slowly, over the course of tens of thousands of years, to fill most of that space.

Meanwhile, about 100,000 years ago in the area now known as the Dirt Belt, another species arose, the Sheragi. The Quarg, by this point, had adopted their policy of building one ringworld in the territory allotted to each species, to make contact with them when the time was right and serve as ambassadors for the Archons. The ringworld in what is now human space was originally meant to be this base in Sheragi territory. The Quarg and the Archons vehemently refused to let the Pug meddle with the Sheragi in any way.

Sheragi are reptilian, flying creatures somewhat resembling dragons. Their species is much more warlike than the Hai. In a period of time that would have seemed like only an eye-blink to the Archons, they went from a pre-spaceflight civilization to one that laid claim to dozens of worlds, before becoming embroiled in a civil war that ended with them driving themselves extinct (as far as the Archons and the Quarg know) with biological weapons. This failure was traumatic for the Archons, in particular, who had failed once more in their directive to broker peace and prevent extinction. It also made them even more bitter toward the Pug, who now adopted a “told-you-so” attitude toward them.

Soon after that, the Hai began fighting civil wars of their own and the Archons, fearing a repeat failure, drove them out of the part of Hai space that is now in human territory, walling them up inside a small enough set of star systems that large divisions couldn’t arise. Instead of stopping the wars, however, this just intensified them, until the entire northern half of Hai space was ravaged by war and most of the planets rendered uninhabitable. Only then did the Hai realize how close they had come to destroying themselves. They retreated to the part of their territory that remained intact, and the Archons sealed them off from the rest of the galaxy. That territory is far too small, however, to keep their species from stagnating.

The civilization boom continues to accelerate. 40,000 years ago the Korath discovered spaceflight. Again, the Quarg made contact with them quickly and did their best to forbid the Pug from getting involved. Korath civilization was peaceful for a relatively long time, enough to approach tier-2 technology.

## The Wanderers
Almost as soon as the Hai left behind their northern territory, the Pug faction that’s been shepherding the Wanderers and using them as an intergalactic cleanup crew moved them into that area, to begin the work of repairing the damage from that war. As of the present they’ve essentially completed their work and the Pug are ready to move them on to cleaning up after the Korath.

The Molt is something akin to a cancer, where some cells in the Wanderer’s body take on a new genetic identity and begins to spread and take over their body. Their immune system will generally fight the transition until the immune system itself is taken over, at which point the immune cells begin to fight the remnants of the old cells.The transition can therefore be made less painful and stressful by using medication to kill off the Wanderer’s immune cells, but they must then be kept in germ-free seclusion until the transition is completed. In the Wanderer story line Iktat Rek will undergo the Molt but choose not to enter seclusion until the very end of the process in order to remain active in leadership of the Wanderer people.

The Molt is triggered when collective stress within the Wanderer population reaches a tipping-point level. It usually, therefore, doesn’t begin until they enter a new territory and are faced with new challenges and unknowns. But it starts earlier in the present case because the Unfettered invasion, as well as contact with human beings, precipitates that same sort of stress and uncertainty.

## The Heliarchs
Meanwhile, a new development south of human space presented the Archons with another problem. The Saryds, Kimek, and Arach developed spaceflight all at nearly the same time (17, 14, and 10 thousand years before the present, respectively). The Archons decided on a new experiment, where they would be limited to relatively small territories but allowed some contact with each other, for the sort of cross-pollination of ideas and experiences that would hopefully prevent the sort of stagnation that the Hai had fallen into. This contact, however, was meant to be tightly limited and controlled by the Quarg.

The Pug had other plans, and secretly fueled a resistance organization of all three species working together to break the yoke of their supposed Quarg overlords. By this point the Archons had seen enough of the Pug’s work firsthand to realize that the Pug might know a thing or two about preventing extinctions, so they took a page out of the Pug playbook: after staging a sham attempt at defending their ringworlds, they withdrew their ships and most of their people from the ringworlds.

The Heliarchs took over the ringworlds, but with no understanding at all of how the technology works. There are entire sections of the ringworlds that they have not even gained access to, yet, and their technology is nowhere near advanced enough to continue the construction of the rings. Instead, for thousands of years, they’ve been “decorating” the ringworlds with solar panels and other architectural warts to make it look like they’re busy continuing the construction. Every once in a while a Quarg ship will enter the system, blast a few of the solar panels off the rings, and engage the Heliarch ships in combat. The Archons have accepted the Pug belief that this will unite the three species behind a common enemy, and the Heliarchs are able to hold onto power because of this external threat.

More importantly, all Quarg ringworlds are joined by portals that allow individuals, cargo, and even entire ships to travel between them instantaneously. (These portals might one day become available to the player in a late stage of the game.) And, there are in fact plenty of Quarg living in the inaccessible parts of the Heliarch ringworlds, maintaining all the machinery, power generators, etc.

The Heliarchs, however, are far from benevolent. The original leaders of the rebellion against the Quarg were motivated by their own story of what happened to the Drak, which went something like this:

The Drak are an advanced civilization who discovered the means, through their technology, to transcend death and pain by combining their bodies and their technology and transforming into space-dwelling creatures known as the Archons. Each Archon carries inside it not just the memories, but the consciousness of entire cities or planets full of Drak individuals. In their new form they have no need of physical resources or territory, yet they have the ability to perceive and to alter the physical world with such fine control that they can even observe and understand the workings of the minds of sentient creatures and insert their own thoughts into those minds. They travel freely between this reality and other hidden dimensions. Their goal is to prevent other species from destroying themselves for long enough that those species, too, can achieve transcendence.

This obsession with immortality led those Heliarch leaders to develop the means to transfer their minds into new bodies. The official story in the Coalition worlds is that every year the Heliarchs select the best and brightest to join them in the work of administering the Coalition, and to some degree this is true. But, some of those who win the athletic competitions, etc. are in fact just being harvested so their bodies can become the next vessel for one of the Heliarch rulers, some of whom have now been alive for thousands of years. (The headbands they wear are stasis field generators designed to shield and preserve their brains if they are attacked and killed.)

A group known as the Resistance (referred to by the Hierarchs as the “14th Resistance,” because the same attempt is made roughly once per century) is trying to overthrow the Hierarchs.

## Humanity
Meanwhile, the human race arose surrounded by the wreckage of two civilizations - the Hai in one direction and the Sheragi in the other. Seeing the territory of the Sheragi uninhabited, in particular, was a constant reminder to the Archons and the Quarg of their failure. So, although it wasn’t a particularly wise decision, they allowed human beings to spread to occupy that entire territory, just so that it would not be left empty and unproductive. The human civil war was the (perhaps) inevitable result of humans being given more space than they could productively govern.

The Archons recognize two different types of species: “primary” species that evolve naturally, and “secondary” species (such as the Archons themselves, the Korath nanobot swarm, the Kor Mereti, and the Alphas) that primary species create. They generally only concern themselves with preserving primary species, unless a primary species goes extinct, in which case they will try to preserve any secondary species they created as well. The official Drak policy on the Alphas is to allow them neither to go extinct nor to gain great power, although the Alphas seem to be self-regulating in that respect - not just because of their reproductive difficulties, but because large groups of Alphas are unable to come together and seek to cooperate for any length of time without ending up killing each other.

At some point during the Alpha wars, an Alpha scientist became frustrated with the question of how the Alphas, despite their obvious superiority, kept getting defeated by ordinary humans. He came to the eventual, thoroughly unwelcome conclusion that it was in fact their capacity for empathy and social cohesion that allowed ordinary human beings to thrive, and the lack of it which drove Alphas over and over again into self-destructive behavior. He believed that the only hope for the survival of the Alphas as a species was therefore to re-engineer themselves yet again into a species with the strength and intelligence of an Alpha but with super-human capacity for empathy and cooperation. Thus, the Betas were not in fact created by humans to defeat the Alphas, but created by an Alpha as a “beta version” of a new human race.

(In addition to Elias Hanover, Commander Nguyen of the Humanitarian Corps was also a Beta, before he was assassinated (not by the Alphas, but by anti-Alpha forces within the Navy - his superhuman speed combined with his inability to bring himself to kill someone, even an Alpha, outed him). Elias’s various aliases over the centuries have mostly been names of Hebrew prophets, e.g. “Elias Hanover” ≈ “Eliyahu Hanavi” = “the prophet Elijah,” and “Jeremy” = “Jeremiah.” That reflects his understanding of his calling in life to stand up against injustice, etc.)

There is, in fact, an Alpha sub-population that believes the Betas are their future, and tries to act entirely based on a morality derived from rational self-interest and logic - sort of a Vulcan culture of repressing their emotions and trying to reason out what actions will best serve their whole community, while ruthlessly punishing any among them who can’t live up to that logical standard. Siding with these “penitent Alphas” could be an alternative story line spanning both tiers 1 and 2 (i.e. parallel to the human and Wanderer story lines, combined).

The Remnant, on the other hand, are biologically human but culturally very different. They are inhabiting a section of the galaxy that, as mentioned previously, has been fallow since the Drak civil wars, and many of their cultural distinctives (such as the tonal language) are an echo of Drak culture, as a result of living in a section of the galaxy “haunted” by the Archons and their memories of the past.

The Questioners are another genetic experiment from the Alpha War era; their abilities are something similar to mirror neuron synesthesia or to the “hyperempathy” of certain characters in Octavia Butler’s Parable of the Sower.

## Korath civil war and raids
The most recent and final Korath civil war began about 2700 AD. In addition to attacking each other, they began raiding Hai space with stolen jump drives. Ironically, this new threat actually led to a sort of cultural renaissance for the previously stagnant Hai. Their species pulled together to drive off the invaders, developing new technology in the process. But when the raids ended (because the Korath had nearly destroyed themselves) the Hai who had found purpose in defending their homeland were suddenly rudderless once more.

The ongoing conflict with the Unfettered is the result of that. Contrary to the Unfettered narrative, there’s actually plenty of Hai who choose to either join the Unfettered or leave them; they are not genetically distinct in any way. The Pug, having their own ideas of what would best serve the Hai, opened the wormhole into human space to give the Hai some other source of excitement less destructive than the Korath raids or the Hai civil war. The Archons, again, have grudgingly allowed this interference.

The Korath, meanwhile, developed more and more destructive weaponry until at last they began destroying entire star systems with supernova bombs; at that point the Archons felt they had no choice but to step in. They gave the Korath an ultimatum, asking them to either allow themselves to be ruled by the Quarg, or be exiled to the Core, to return only once they showed signs of developing wisdom.

The Kor Mereti drones are autonomous, guided by a collective artificial intelligence. The Kor Sestor drones are guided remotely by a small number of Korath hiding out in bunkers deep underground on ruined planets. (Up until the Alphas infiltrate the bunkers, kill them, and take over control of the fleet.) This represents the different philosophies of the two factions (one valuing top-down hierarchical control and the other thinking more collectively).

The Alphas, meanwhile, have several separate enclaves throughout the galaxy, one of which has been stealing jump drives from the Korath exiles and selling them to the Unfettered Hai in exchange for weaponry. (“Prometheus stole fire from the gods, and was punished. We steal fire from Prometheus, which is much safer.”)

Danforth’s antimatter bomb is a weapon stolen from the Korath. The Syndicate cloaking device was stolen from the Korath, who in turn stole it from someone else (probably the Pug). (Because the Syndicate have a few jump drives, they’ve also contacted the Hai and begun reverse-engineering their technology.)

## Current status
* Wanderers: Friendly with Pug, avoided by the Archons. Stable and prosperous.
* Sheragi: Pug were prevented from meddling with them. Went (nearly) extinct.
* Hai: Friends with Quarg. No known Pug involvement. Stable, but stagnant.
* Korath: Friends with Quarg. Pug avoid their territory. Their civilization is now in ruins.
* Coalition: Drove out the Quarg, with Pug help. Prosperous society.
* Humans: Friends with Quarg. Some Pug meddling. Fractured, unstable society.

## Current population sizes:
* Humanity: 100 B
* Hai: 40 B
* Wanderers: 1 B
* Korath: 1 B (mostly the Kor Efret)
* Saryds: 30 B
* Kimek: 200 B
* Arachi: 50 B
* Pug: unknown; at least 2-3 B in our galaxy.
* Quarg: unknown; >1,000 B per complete ringworld.

## Alien languages
In the Hai language, words tend to be short, soft, and fluid, with a preponderance of vowels and diphthongs. Multiple consonants seldom appear next to each other. There are no articles. In translation, fluent speakers may be indistinguishable from human speech; less fluent speakers will often omit articles, or insert them where they are not needed. (Culturally, the original design document describes the Hai as “perpetually stoned squirrels.” The fact that their star systems have names that sound like “Du-ude,” “Far out,” “Awesome,” “Hey dude,” “YOLO,” and “Radical” is just because thinking up lots of alien words all at once is hard. :))

In the Wanderer language, words tend to be long, with lots of velar and guttural sounds and fewer dental sounds, due to their anatomy (beaks instead of teeth). When two vowel sounds appear next to each other they are separated by a glottal stop (transcribed as an apostrophe) rather than being combined into a diphthong. Because Wanderer speech is translated for you via a computer and there is not always an exact match, it sometimes contains multiple possible translations in brackets.
Korath language is somewhat similar to German, with lots of hard consonantal sounds and long, compound words. Because they are reptiles, their speech contains a lot of sibilant sounds. (Side note: the names of the Korath core worlds were invented very early in the game. Then, after I added the Wanderers and their language that’s full of apostrophes (glottal stops), I realized that I’d made the two languages too similar. The more recently added Korath systems have names I chose to sound more like a sibilant, reptilian language, but it was too late to change those core systems that were already in everyone’s save files. The Kor Sestor ship names are actually letters of their alphabet, e.g. Kar Ik Vot ≈ K.I.V., Tek Far Osk = T.F.S., etc.)

Archons communicate telepathically, meaning that they can speak fluently to an individual of any other species. Their “native” language is therefore never represented in the game, and the names they choose to go by are taken from the mythology of other species. (“Sword of Eden” being obviously a reference to a human fall narrative, and “Snows of Far Door Nai” and “Kasiliri's Folly” being references to similar stories in Hai and Korath culture, respectively.) With creatures like the Archons the distinction between natural and supernatural gets a bit murky, but their “psychic” abilities are actually because they’re powerful and precise enough to observe and affect the electrical currents in your brain from a distance.

**Comment by Zitchas** [comment]
Thanks for posting! Definitely a good thing to have on here.

**Reply by MasterOfGrey** [reply]
Do you wanna pin this in the category?
